.github/workflows/issues.yml

```diff
@@ -11,27 +11,26 @@ env:
   SUPPORT_LBL: pending::support
 
 jobs:
-  # NOTE: doesn't catch cases where multiple users act as the author/reporter,
-  # this is just an effort to catch the majority of support cases
+  # NOTE: will update label if anyone responds, not just the author/reporter
   # TODO: create conda-issue-sorting team and modify this to toggle label based on
   # whether a non-issue-sorting engineer commented
   pending_support:
-    # if [pending::feedback] and the author responds
+    # if [pending::feedback] and anyone responds
     if: >-
       !github.event.repository.fork
       && github.event_name == 'issue_comment'
       && github.event.action == 'created'
```

Comment:
Let's also remove these two checks, they're redundant since that's the only condition in which this workflow will run in the first place (they're remnants from a time when this workflow was part of a larger process)

```suggestion
```